app/assets/stylesheets/custom.css

```diff
@@ -24,3 +24,18 @@ nav {
 .home_blurb {
   text-align: left;
 }
+
+.center {text-align: center; margin-left: auto; margin-right: auto; margin-bottom: auto; margin-top: auto;}
+
+.right {text-align: right; margin-left: auto; margin-right: auto; margin-bottom: auto; margin-top: auto;}
+
+img {
```

Comment:
will have to be altered so the show page displays images larger than a thumb nail